pkg/registry/core/pod/storage/eviction_test.go

```diff
@@ -242,8 +243,11 @@ func TestEvictionDryRun(t *testing.T) {
 			evictionRest := newEvictionStorage(storage.Store, client.PolicyV1beta1())
 			eviction := &policy.Eviction{ObjectMeta: metav1.ObjectMeta{Name: "foo", Namespace: "default"}, DeleteOptions: tc.evictionOptions}
 			_, err := evictionRest.Create(testContext, pod.Name, eviction, nil, tc.requestOptions)
-			if err != nil {
-				t.Fatalf("Failed to run eviction: %v", err)
+			if err == nil {
+				t.Fatalf("should have received error")
+			}
+			if !strings.Contains(err.Error(), "no kind is registered for") {
+				t.Fatalf("should have received 'no kind is registered'")
```

Comment:
is this really the error that we're supposed to receive? That seems odd.

Reply:
Here is the complete error:
```
 no kind is registered for the type core.Pod in scheme "k8s.io/kubernetes/pkg/registry/core/pod/storage/eviction_test.go:171"
```
I can modify the assertion based on feedback.

Reply:
I don't think this error is relevant or what we want to test. It probably shouldn't even happen.

Reply:
I will remove the check for error text in the next push.

The error is genuine - as 'no kind is registered for the type core.Pod in scheme' showed

Reply:
Agreed, you might have found something wrong with this test, you sort of have to fix it now :-/